src/ucp/core/ucp_worker.c

```diff
@@ -1100,16 +1109,31 @@ ucs_status_t ucp_worker_iface_open(ucp_worker_h worker, ucp_rsc_index_t tl_id,
     uct_config_release(iface_config);
 
     if (status != UCS_OK) {
-       return status;
+       goto err_free_iface;
+    }
+
+    VALGRIND_MAKE_MEM_UNDEFINED(&wiface->attr, sizeof(wiface->attr));
```

Comment:
maybe use ucs_malloc in line 1058 instead?

Reply:
Why? i'd better init iface with zeros

Reply:
this way valgrind will notify about any uninitialized field